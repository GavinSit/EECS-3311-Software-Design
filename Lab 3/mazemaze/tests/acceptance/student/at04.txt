--test abort
new_game(hard)
abort 
new_game(medium)
abort
new_game(easy)
abort
