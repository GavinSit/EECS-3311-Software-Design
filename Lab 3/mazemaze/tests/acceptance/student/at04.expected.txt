  State: 0 -> ok
->new_game(hard)
  State: 1 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
  | X                     |               |       |           |
  +---+   +---+---+---+   +---+---+   +   +---+   +---+---+   +
  |                   |       |       |           |   |       |
  +---+   +   +---+   +   +   +   +   +---+   +   +   +---+   +
  |   |   |           |   |           |       |           |   |
  +   +---+   +---+   +   +---+   +---+   +---+   +---+   +---+
  |               |       |       |   |   |   |   |   |   |   |
  +   +   +   +   +   +   +   +   +   +---+   +---+   +   +   +
  |       |   |   |   |       |       |   |                   |
  +   +---+---+---+---+---+---+   +---+   +   +   +   +   +   +
  |                       |           |       |   |       |   |
  +---+---+   +   +---+   +---+---+   +---+   +   +   +   +---+
  |               |               |           |       |       |
  +---+   +   +---+---+   +---+---+---+---+   +---+---+   +---+
  |       |   |   |   |               |                   |   |
  +---+---+   +   +   +   +   +---+   +---+   +   +   +---+   +
  |       |                                   |   |   |   |   |
  +   +---+---+   +---+---+---+---+   +   +   +   +   +   +   +
  |       |   |   |       |   |       |       |               |
  +---+   +   +   +---+   +   +   +---+   +   +   +---+   +---+
  |                       |       |                           |
  +---+---+   +---+   +   +---+---+   +   +   +---+   +   +   +
  |   |   |   |               |   |   |       |       |   |   |
  +   +   +---+---+---+---+   +   +---+   +   +   +   +---+   +
  |           |   |   |                           |   |       |
  +---+   +   +   +   +   +   +---+   +---+   +   +   +---+   +
  |       |           |       |           |   |       |       |
  +   +---+   +   +   +   +---+---+---+   +---+   +   +   +   +
  |           |                   |           |   |   |       |
  +---+---+---+---+---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 3

->abort
  State: 2 -> ok
->new_game(medium)
  State: 3 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X         |   |                           |
  +   +   +   +   +   +---+   +   +---+---+---+
  |   |   |                               |   |
  +   +---+---+   +---+---+---+---+   +   +   +
  |   |   |       |       |   |   |   |   |   |
  +---+   +   +---+---+   +   +   +---+---+   +
  |   |                                   |   |
  +   +---+   +---+---+---+   +---+   +   +   +
  |           |       |       |   |       |   |
  +   +   +---+---+   +---+   +   +---+   +   +
  |           |           |                   |
  +   +   +---+   +---+   +---+   +---+---+   +
  |           |       |   |   |   |           |
  +---+   +   +---+   +   +   +---+---+---+   +
  |       |       |           |   |       |   |
  +   +   +   +---+   +   +---+   +---+   +   +
  |   |   |   |       |   |   |       |       |
  +   +   +   +   +   +   +   +   +   +   +   +
  |               |               |       |   |
  +---+   +   +   +---+   +   +---+---+   +   +
  |                   |       |               |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 2
  Score: 0 / 5

->abort
  State: 4 -> ok
->new_game(easy)
  State: 5 -> ok
  +-*-+---+---+---+---+---+---+
  | X         |       |       |
  +   +---+   +   +   +---+   +
  |               |   |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +---+   +---+---+   +---+   +
  |   |       |   |   |       |
  +   +   +   +   +   +   +   +
  |           |               |
  +   +---+   +---+   +   +   +
  |   |       |   |   |       |
  +---+   +---+   +   +   +---+
  |                           |
  +---+---+---+---+---+---+- -+

  Game Number: 3
  Score: 0 / 6

->abort
  State: 6 -> ok
